docs/src/pages/style/typography/DeprecatedTypes.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  root: {
+    width: '100%',
+    maxWidth: 500,
+  },
+};
+
+function Types(props) {
+  const { classes } = props;
+
+  return (
+    <div className={classes.root}>
+      <Typography variant="headline1" gutterBottom suppressDeprecationWarnings>
```

Comment:
`suppressDeprecationWarnings` is a dead property right?

Reply:
It is, will fix that and look if I forget anything else.

Reply:
That hole section was conceptually wrong. I wanted to display deprecated types but failed to do so in any way. It is now displaying deprecated and restyled variants with a v1 typography theme.